sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/ConfigurationSettingsFilter.Serialization.cs

```diff
@@ -18,13 +19,24 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("label");
                 writer.WriteStringValue(Label);
             }
+            if (Optional.IsCollectionDefined(Tags))
+            {
+                writer.WritePropertyName("tags");
+                writer.WriteStartArray();
+                foreach (var item in Tags)
```

Comment:
why item and not tag? Also, we dont use var unless it's absolutely clear what type it is, which amounts to only when we new instances.

Reply:
VAR all the things!